sklearn/feature_extraction/text.py

```diff
@@ -1511,14 +1511,15 @@ class TfidfTransformer(_OneToOneFeatureMixin, TransformerMixin, BaseEstimator):
 
     Parameters
     ----------
-    norm : {'l1', 'l2'}, default='l2'
+    norm : {'l1', 'l2', 'None'}, default='l2'
```

Comment:
```suggestion
    norm : {'l1', 'l2'} or None, default='l2'
```

Reply:
This is not a string but `None` the Python value.

Reply:
done in 97b01d7